src/Microsoft.Health.Fhir.Liquid.Converter/Hl7v2/Hl7v2DataParser.cs

```diff
@@ -49,7 +49,7 @@ public Hl7v2Data Parse(string message)
             return result;
         }
 
-        private List<Hl7v2Field> ParseFields(string dataString, Hl7v2EncodingCharacters encodingCharacters, bool isHeaderSegment = false)
+        private static List<Hl7v2Field> ParseFields(string dataString, Hl7v2EncodingCharacters encodingCharacters, bool isHeaderSegment = false)
```

Comment:
too many static methods, consider if this class has mutable status or just make the whole class as static?

Reply:
Good point. The data parsers are made static.